templates/artist/includes/artist_detail.html

```diff
@@ -0,0 +1,6 @@
+{% load thumbnail %}
```

Comment:
Now that the thumbnail code was moved to `templates/artist/artist_detail.html`, this whole file can be deleted.